scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -3002,12 +3003,26 @@ def setup(params):
       formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
   # Sets environment variables for MKL
+  # If OS ENV variables are overridden, a warning msg is printed.
   if params.mkl:
+    if '-kmp_blocktime' not in str(sys.argv) and 'KMP_BLOCKTIME' in os.environ:
```

Comment:
Checking sys.argv doesn't work in all cases, since you can also specify this with two dashs: `--kmp_blocktime`. Instead, you can check if flags are present with:
```
absl_flags.FLAGS.is_parsed() and absl_flags.FLAGS['kmp_blocktime'].present
```
Maybe also wrap that code fragement in a function to check if a flag is present.

Reply:
Actually, the code fragment should be:

```
absl_flags.FLAGS.is_parsed() and 'kmp_blocktime' in absl_flags.FLAGS and absl_flags.FLAGS['kmp_blocktime'].present
```

since the flags aren't always defined in tests.